test/e2e/helpers_test.go

```diff
@@ -387,6 +388,19 @@ func verifyPostgres(ctx context.Context, t *testing.T, env environments.Environm
 	require.GreaterOrEqual(t, migrationJob.Status.Succeeded, int32(1))
 }
 
+// containConfigPushCountMetric check if the prometheus metrics contains metrics for pushing configurations,
+// which indicate that this instance is the leader.
+func containConfigPushCountMetric(t *testing.T, url string, client *http.Client) bool {
```

Comment:
How about changing the function like this:
- `GET` from an URL
- returns true if status code is 200 and response contains certain string
So that this function could be more general and could be used in more places.